.settings.xml

```diff
@@ -46,6 +46,10 @@
 				</repository>
 			</repositories>
 			<pluginRepositories>
+				<pluginRepository>
```

Comment:
I am not sure why, but if I put this one here explicitly, it will not go to spring repositories for download, but use maven central